packages/video_player/video_player/lib/video_player.dart

```diff
@@ -153,7 +153,11 @@ class VideoPlayerValue {
       isBuffering: isBuffering ?? this.isBuffering,
       volume: volume ?? this.volume,
       playbackSpeed: playbackSpeed ?? this.playbackSpeed,
-      errorDescription: errorDescription ?? this.errorDescription,
+      errorDescription: errorDescription == null
```

Comment:
If this is right, it definitely needs a comment explaining it, because it looks incorrect to me. The expected behavior of a `copyWith` is that anything not explicitly passed as an override is preserved.